homeworks/nazar_kostiv/pyats01_hw.py

```diff
@@ -3,46 +3,34 @@
 from pyats.aetest import Testcase, test, main
 from calculation import add, divide
 
-parameters = {
-        'num1': 3,
-        'num2': 0
-    }
 
 class SmokeTest(Testcase):
-    parameters = {
-        'num1': 3,
-        'num2': 0
-    }
 
     @test
     def test_add(self, steps, num1, num2):
-        with steps.start('Add') as step:
-            assert add(num1, num2) == num1 + num2
         with steps.start('Result bigger then 0') as step:
-            try:
-                assert add(num1, num2) > 0
-            except AssertionError:
-                self.skipped('Result is less then 0)')
-
-
-
+            if add(num1, num2) < 0:
+                step.skipped('Result is less then 0)')
 
     @test
     def test_devide(self, steps, num1, num2):
```

Comment:
"self" is not used, so is not required
```suggestion
    def test_devide(steps, num1, num2):
```